### ISSUE TITLE
(rc1) menu/addons: Incorrect Listing with multiple Addons 

### ISSUE THREAD

**Issue by lyseah** [OpenCollarTeam/OpenCollar#451]
**What version of OpenCollar are you using?**
OpenCollar 8.0 - Release Candidate 1

**What behavior did you expect?**
When adding more than one addon, it should be available in the Addons Menu.
The list item should have the addon name and should be clickable to open the specific addons menu.

**What behavior did you see instead?**
The first list index is displayed correctly, while other addons are listed by Object Name instead of Addon Name. Their menu button is also not triggering the specific menu, rendering the addon inaccessible.

**What steps does someone need to take to reproduce the problem?**

1. Create two objects with different names and place oc_addon_template.lsl inside them. (In my example "# 1 Object" & "# 2 Object")
2. Modify the addon_name inside to something unique. (In my example "First Addon" & "Second Addon")
3. Wear or rez them, so they authenticate with the collar.

![image](https://user-images.githubusercontent.com/76599906/103153532-78f3b680-4791-11eb-9646-4a823150b966.png)

**Comment by lyseah** [comment]
Also, adding an llOwnerSay to g_lAddons looks like it's borking up the optin value (\xEF\xB7\x90) if its empty?
```
b1e9a268-f8cf-9f70-37fa-78c8fdb5134c~First Addon~1608994815~﷐~87e5c845-bda0-d249-1c02-61c7f6d08e12~Second Addon~1608994819~﷐
```
which is also the "name" displayed for index 2 in the screenshot above. So it looks like the striding is a bit broken?

**Comment by zontreck** [comment]
Fixed by #452
